### ISSUE TITLE
Cannot play a sound more than once, simultaneously

### ISSUE THREAD

**Issue by jgranick** [goldfire/howler.js#682]
I could be mistaken, but after loading a sound, it seems that you cannot play it more than once simultaneously

```js
howl.play ();

setTimeout (function () {
    howl.play ();
}, 1000);
```

Am I misunderstanding how the API works?

**Comment by goldfire** [comment]
Can you give some more details and/or a full example? Playing sounds simultaneously is one of the core features of howler, and I'm not seeing any issues with it on my end.

**Comment by jgranick** [comment]
Thank you for the help, and for the library :smile:

I just found that actually, Howler was working _too well_ :flushed:

I had a `play` method which handles resuming a paused sound. After further digging, I found that the code was running something like this:

```js
function play () {
    var time = howl.seek (id);
    id = howl.play ();
    howl.seek (time, id);
}
```

The issue is obvious -- `id` has not been set, yet, when it is played the first time. Since it is undefined, it is probably behaving similar to `howl.seek (0);`

As a result, the second sound will seek time of ID `0`, which in case appears to have been the first sound. That means that no matter what size delay I used, the second sound seeked to play back from the same spot. Howler was properly playing both sounds at the exact same time so it sounded like one sound only 😖 

Thank you again
